packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_world_map_card.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+import 'package:openfoodfacts/model/KnowledgePanelElement.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+class KnowledgePanelWorldMapCard extends StatelessWidget {
+  const KnowledgePanelWorldMapCard(this.mapElement);
+
+  final KnowledgePanelWorldMapElement mapElement;
+
+  @override
+  Widget build(BuildContext context) {
+    if (mapElement.pointers.isEmpty || mapElement.pointers.first.geo == null) {
+      return EMPTY_WIDGET;
+    }
+    return Padding(
+      padding: const EdgeInsets.only(bottom: MEDIUM_SPACE),
+      child: SizedBox(
+        height: 200,
+        child: FlutterMap(
+          options: MapOptions(
+            // The first pointer is used as the center of the map.
+            center: LatLng(
+              mapElement.pointers.first.geo!.lat,
+              mapElement.pointers.first.geo!.lng,
+            ),
+            zoom: 13.0,
+          ),
```

Comment:
The map should include all `Marker`s, and the zoom should be computed accordingly - with a default for single Marker maps of course.

Reply:
So I talked to Stephane about this, I need to pass in either the `center` or `bounds`, bounds would need two corners (unfortunately there's no way of passing all pointers and expecting the map to display that). Stephane says for now, it's ok to just center on the first pointer.

Reply:
Understood. I'll take care of that later.

Reply:
The fix will probably involve:
```dart
LatLngBounds.fromPoints(List<LatLng> points)

// ... and ...

mapController.fitBounds(LatLngBounds bounds)
```